index.html

```diff
@@ -54,34 +55,32 @@
         doorTwoChart = new google.visualization.Gauge(document.getElementById("doorTwoChart"));
         noiseChart = new google.visualization.Gauge(document.getElementById("noiseChart"));
         socket.on('onlineUserCount', function (msg) {
+
           $('#onlineCounter').html(msg);
           onlineUserCountCachedData = msg;
           let chartData = [["Label", "Value"], ["", msg]];
           drawGaugeChart("usersOnlineChart", chartData);
         });
         socket.on('doorOne', function (doorOneAverage, doorOnePassedThroughToday) {
-          let doorOneAvgInteger = Math.floor(doorOneAverage);
-          doorOneCachedAverage = doorOneAvgInteger;
+          if (doorOneAverage > max) {
+            max = doorOneAverage;
+          }
+          doorOneCachedAverage = doorOneAverage;
           $('#doorOne').html(doorOnePassedThroughToday);
-          let chartData = [["Label", "Value"], ["Door one", doorOneAvgInteger]];
```

Comment:
Here (inside the gauge) we use the integer instead of the real number received from the server because of requirements.